client/my-sites/plugins/categories/use-categories.tsx

```diff
@@ -379,6 +419,338 @@ export const getCategories: () => Record< string, Category > = () => ( {
 			},
 		],
 	},
+	affiliate: {
+		menu: __( 'Affiliate' ),
+		title: __( 'Affiliate' ),
+		slug: 'affiliate',
+		tags: [ 'affiliate', 'affiliates', 'affiliate marketing' ],
+		preview: [],
+	},
+	quiz: {
+		menu: __( 'Quiz' ),
+		title: __( 'Quiz' ),
+		slug: 'quiz',
+		tags: [ 'quiz', 'questionnaire', 'exam', 'personality quizzes' ],
+		preview: [],
+	},
+	landingPage: {
+		menu: __( 'Landing Page' ),
+		title: __( 'Landing Page' ),
+		slug: 'landing-page',
+		tags: [ 'landing page', 'page builder' ],
+		preview: [],
+	},
```

Comment:
Only for the Landing Page option, the category title on top of the results shows the slug instead of the title:
![CleanShot 2023-05-05 at 11 13 37@2x](https://user-images.githubusercontent.com/11555574/236433001-a67877e2-47ee-48aa-840f-78661f2b5d8f.png)

I don't see any issues with this config and also couldn't see why it would show the slug here:
https://github.com/Automattic/wp-calypso/blob/3743b983a2422bdc50b848811b9b3b87abbbb614/client/my-sites/plugins/plugins-category-results-page/index.jsx#L16

Reply:
Thank you very much for the thorough review! I am going to fix the `landingPage`, it is probably related with the `key` of the object being different to the `slug`